modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/services/Products.js

```diff
@@ -37,3 +37,19 @@ export function getChannelId(channelName) {
 		`${headlessAPI}/channels?filter=contains(name, '${channelName}')&fields=id`
 	);
 }
+
+export function getProductsNames() {
```

Comment:
I think it to be better to change the function name.

Does it make sense to you?

```suggestion
export function getProductNames() {
```